surf_trashbin/js/app.js

```diff
@@ -27,7 +27,12 @@ $(document).ready(function() {
 						return;
 					}
 					this._initialized = true;
-					asdasdasd(groupName);
```

Comment:
Good thing we're removing this line from production code :) Are there more lines like this anywhere else in the codebase?

Reply:
Indeed! 😄 no it is only here!